src/libs/Notification/PushNotification/subscribeToPushNotifications.ts

```diff
@@ -33,6 +33,9 @@ Onyx.connect({
 
             PushNotification.onReceived(PushNotification.TYPE.REPORT_ACTION, applyOnyxData);
             PushNotification.onSelected(PushNotification.TYPE.REPORT_ACTION, navigateToReport);
+
+            PushNotification.onReceived(PushNotification.TYPE.TRANSACTION, applyOnyxData);
+            PushNotification.onSelected(PushNotification.TYPE.TRANSACTION, navigateToReport);
```

Comment:
NAB: We don't have to refactor or fix this now but it's kind of weird that we do the same thing for all of these different types.

Reply:
Agreed. I'm working on removing the moneyRequest type since it's redundant. Handling transactions will also change a bit to handle some edge cases. I'm just handling the simple case for now though.